CS342MemberViews/Sources/EmmyThamakaison/BubbleView.swift

```diff
@@ -0,0 +1,58 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct IdentifiableFact: Identifiable {
+    var id = UUID()
+    var fact: String
+}
+
+struct BubbleView: View {
+    
+    let facts = [
```

Comment:
Very cool that you have separated the facts out into a model type that you use in your view, very good solution! 👍